Dockerfile

```diff
@@ -17,4 +17,6 @@ WORKDIR /tmp/connect
 RUN bundle -j8
 ADD . /tmp/connect
 RUN chown -R nobody /tmp/connect
-RUN chmod +x /tmp/connect/docker/runall.sh
+RUN chmod +x /tmp/connect/docker/prophet_tests.sh /tmp/connect/docker/integration.sh
+ADD .oscrc /root/.oscrc
+ADD .regcode /root/.regcode
```

Comment:
Do you mean why don't we fetch them from gaffer at runtime inside the image? or why don't we RUN wget to get them? The second option is cached too aggressively by docker build.